installers/rpm-code/dnf-download.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set +euo pipefail
+source ~/.bashrc
+set -euo pipefail
+source ./dev-env.sh
+declare f
+sudo dnf makecache
+declare d=dnf-download-tmp
+rm -rf "$d"
+mkdir "$d"
+declare common=0
+for f in "$@"; do
+    dnf download --downloaddir "$d" "rscode-$f"
+    if [[ $1 == common ]]; then
+        common=1
+        cd ~/src/radiasoft/container-rpm-code
+        if [[ ! $(docker images | grep radiasoft/fedora) ]]; then
+            cd container-fedora
+            # in case set by dev-env.sh, because server isn't running yet
+            radia_run container-build
+        fi
+        radia_run container-build
+    fi
+done
+mv "$d"/*.rpm "$rpm_code_install_dir"
```

Comment:
What is the purpose of downloading to dnf-download-tmp instead of $rpm_code_install_dir?

Reply:
2ac0988